### PR TITLE
ocrvs-312 Change default notification to floating

### PR DESCRIPTION
**Pull request by sadmananik**
ocrvs-312

### CODE REVIEW

File: packages/client/src/views/Settings/ChangePhonePage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+  it('Continue button should be disabled', () => {
+    expect(
+      component.find('#continue-button').hostNodes().prop('disabled')
+    ).toBe(true)
+  })
+
+  it('should enable continue button when input valid phone number', async () => {
+    const textInput = component.find('#PhoneNumber').hostNodes()
+    expect(textInput).toHaveLength(1)
```
Comment (Zangetsu101) [inline packages/client/src/views/Settings/ChangePhonePage.test.tsx:65]: Is this required here?

File: packages/client/src/views/Settings/ChangePhonePage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    const VerifyTextInput = component.find('#verifyCode').hostNodes()
+    expect(VerifyTextInput).toHaveLength(1)
+    component.find('input').simulate('change', {
+      target: { name: 'verifyCode', value: '000000' }
+    })
+    expect(component.find('#verify-button').hostNodes().prop('disabled')).toBe(
+      false
+    )
+    component.find('#verify-button').hostNodes().simulate('click')
+    expect(history.location.pathname).toContain('/')
```
Comment (Zangetsu101) [inline packages/client/src/views/Settings/ChangePhonePage.test.tsx:114]:
```suggestion
    component.update()
    expect(history.location.pathname).toContain('/settings')
```

File: packages/client/src/views/Settings/ChangePhonePage.test.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  })
+
+  it('should enable continue button when input valid phone number', async () => {
+    const textInput = component.find('#PhoneNumber').hostNodes()
+    expect(textInput).toHaveLength(1)
+
+    component.find('input').simulate('change', {
+      target: { name: 'PhoneNumber', value: '01741234567' }
+    })
+    await waitForElement(component, '#continue-button')
```
Comment (Zangetsu101) [inline packages/client/src/views/Settings/ChangePhonePage.test.tsx:70]: `#continue-button` should already be in the DOM, so I don't think we need to wait for it?

File: packages/client/src/views/Settings/ChangePhonePage.test.tsx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    component.update()
+    component.find('#continue-button').hostNodes().simulate('click')
+    expect(component.find('#verify-button').hostNodes().prop('disabled')).toBe(
+      true
+    )
+    const VerifyTextInput = component.find('#verifyCode').hostNodes()
+    expect(VerifyTextInput).toHaveLength(1)
```
Comment (Zangetsu101) [inline packages/client/src/views/Settings/ChangePhonePage.test.tsx:106]: This seems unnecessary here

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: packages/client/src/views/Settings/ChangePhonePage.test.tsx (hunk 2 above)
Comment (sadmananik) [reply packages/client/src/views/Settings/ChangePhonePage.test.tsx:114]: Changed.

File: packages/client/src/views/Settings/ChangePhonePage.test.tsx (hunk 4 above)
Comment (sadmananik) [reply packages/client/src/views/Settings/ChangePhonePage.test.tsx:106]: Removed.

File: packages/client/src/views/Settings/ChangePhonePage.test.tsx (hunk 3 above)
Comment (sadmananik) [reply packages/client/src/views/Settings/ChangePhonePage.test.tsx:70]: Removed

File: packages/client/src/views/Settings/ChangePhonePage.test.tsx (hunk 1 above)
Comment (sadmananik) [reply packages/client/src/views/Settings/ChangePhonePage.test.tsx:65]: Removed

**Review by emon-swe-sust — APPROVED** [review]

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**